README.md

```diff
@@ -19,119 +23,123 @@ Trader is an autonomous service that performs bets on existing prediction market
+export OMEN_CREATORS='["0x89c5cc945dd550BcFfb72Fe42BfF002429F46Fec"]'
+
+export BET_AMOUNT_PER_THRESHOLD_000=0
+export BET_AMOUNT_PER_THRESHOLD_010=0
+export BET_AMOUNT_PER_THRESHOLD_020=0
+export BET_AMOUNT_PER_THRESHOLD_030=0
+export BET_AMOUNT_PER_THRESHOLD_040=0
+export BET_AMOUNT_PER_THRESHOLD_050=0
+export BET_AMOUNT_PER_THRESHOLD_060=30000000000000000
+export BET_AMOUNT_PER_THRESHOLD_070=40000000000000000
+export BET_AMOUNT_PER_THRESHOLD_080=60000000000000000
+export BET_AMOUNT_PER_THRESHOLD_090=80000000000000000
+export BET_AMOUNT_PER_THRESHOLD_100=100000000000000000
+
+export BET_THRESHOLD=5000000000000000
+
+export PROMPT_TEMPLATE='With the given question "@{question}" and the `yes` option represented by `@{yes}` and the `no` option represented by `@{no}`, what are the respective probabilities of `p_yes` and `p_no` occurring?'
+```
+
+These are the description of the variables used by the Trader service:
+
+- `RPC_0`: RPC endpoint for the agent (you can get an RPC endpoint, e.g. [here](https://getblock.io/)).
+- `CHAIN_ID`: identifier of the chain on which the service is running (Gnosis=100).
+- `ALL_PARTICIPANTS`: list of all the agent addresses participating in the service. In this example we only are using a single agent.
+- `SAFE_CONTRACT_ADDRESS`: address of the agents multisig wallet created [in the previous section](#prepare-the-keys-and-the-safe).
+- `OMEN_CREATORS`: addresses of the market creator(s) that the service will track
+  for placing bets on Omen. The address `0x89c5cc945dd550BcFfb72Fe42BfF002429F46Fec` corresponds to the Market creator agent for the Hackathon.
+- `BET_AMOUNT_PER_THRESHOLD_X`: amount (wei) to bet when the prediction returned by the AI Mech surpasses a threshold of `X`% confidence for a given prediction market. In the values provided above the amounts vary between 0.03 xDAI (60% confidence) and 0.1 xDAI (100% confidence).
+- `BET_THRESHOLD`: threshold (wei) for placing a bet. A bet will only be placed if `expected_return - bet_fees >= BET_THRESHOLD`. [See below](#some-notes-on-the-service).
+- `PROMPT_TEMPLATE`: prompt to be used with the prediction AI Mech. Please keep it as a single line including the placeholders `@{question}`, `@{yes}` and `@{no}`.
+
+
+## Run the service
+Once you have configured (exported) the environment variables, you are in position to run the service.
+
+- Fetch the service:
 
     ```bash
     autonomy fetch --local --service valory/trader && cd trader
     ```
 
-- Build the image:
+- Build the Docker image:
 
     ```bash
     autonomy build-image
     ```
 
-- Prepare the agent keys for running the service with a single agent:
+- Copy your `keys.json` file prepared [in the previous section](#prepare-the-keys-and-the-safe) in the same directory:
 
     ```bash
-    cat > keys.json << EOF
-    [
-    {
-        "address": "<your_agent_address>",
-        "private_key": "<your_agent_private_key>"
-    }
-    ]
-    EOF
+    cp path/to/keys.json .
```

Comment:
Why add an extra step?

Reply:
There is no extra step. All related to "keys set up" (including safe, etc...) has been moved to a separate section, in case we wanto to make some script to reduce that section.

Reply:
This was done to prepare the key file and safe in a previous section (1 time process) and running the agent in another section. 
It is an extra step, but it simplifies the "running the agent" section. Also, we can include a script to execute all key and safe generation in a script, and reduce that section.

Reply:
> Also, we can include a script to execute all key and safe generation in a script, and reduce that section.

Yes, that would be the best option.